commands/displayers/apps.go

```diff
@@ -502,3 +502,42 @@ func (b Buildpacks) JSON(w io.Writer) error {
 	e.SetIndent("", "  ")
 	return e.Encode(b)
 }
+
+type AppInstances []*godo.AppInstance
+
+var _ Displayable = (*AppInstances)(nil)
+
+func (a AppInstances) Cols() []string {
+	return []string{
+		"ComponentName",
+		"ComponentType",
+		"InstanceName",
```

Comment:
Since we are listing the instances here, I think instance name should be the first column. Also, `InstanceName` can be changed to `Name` I think. 

I feel like this is the convention followed for other commands in doctl (and commands in kubectl for example). But I'm not entirely sure if what I'm saying is right, so feel free to use your own judgement.

```suggestion
		"Name",
		"ComponentName",
		"ComponentType",
```

Reply:
I agree and went forward with the suggested changes.